jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/operation/IndividualGenerateTest.java

```diff
@@ -1,14 +0,0 @@
-package com.cerner.jwala.ui.selenium.operation;
```

Comment:
Do we need to remove this ?

Reply:
this is also included individual control operations